src/inmanta/module.py

```diff
@@ -1666,7 +1665,16 @@ def get_relation_precedence_rules(self) -> List[RelationPrecedenceRule]:
         return [RelationPrecedenceRule.from_string(rule_as_str) for rule_as_str in self.relation_precedence_policy]
 
     def get_index_urls(self) -> List[str]:
-        return self.pip.index_url
+        """
+        Once setting repos with type package is no longer supported, this method can return self.pip.index_url alone.
+        """
```

Comment:
A docstring is meant for the users of the method. This message is meant for the maintainers of the method.
```suggestion
        # Once setting repos with type package is no longer supported, this method can return self.pip.index_url alone.
```